code/nick/python/lab_8_pick6.py

```diff
@@ -78,10 +77,9 @@ def pick6(tickets):
 # Version 2
 # The ROI (return on investment) is defined as (earnings - expenses)/expenses. 
 # Calculate your ROI, print it out along with your earnings and expenses.
-            roi = ((earning - (expense * -1) / (expense * -1) * 100))
+            roi = round((((earning - (expense * -1)) / (expense * -1) * 100)),2)
```

Comment:
ROI still not functioning quite right. All the pieces are there, take another look at the example equation.